src/dndbeyond/content-scripts/character.js

```diff
@@ -898,6 +898,12 @@ function rollAction(paneClass, force_to_hit_only = false, force_damages_only = f
             settings_to_change["charger-feat"] = false;
         }
 
+        // Druid: Circle of Spores - Symbiotic Entity
+        if (isMeleeAttac && character.hasClassFeature("Symbiotic Entity") && character.getSetting("druid-symbiotic-entity", false)) {
```

Comment:
You clearly didn't test this 🙄  (type in variable name 😝)

Reply:
Not sure what you're talking about 😇